nestedtensor/nested/monkey_patch.py

```diff
@@ -134,6 +134,14 @@ def new_fn(self):
         setattr(NestedTensor, function_name + "_", _gen_fn(function_name + "_"))
         C_functions[getattr(torch, function_name)] = function_name
     # <
+    
+    # > NN functions supported by _C
+    for function_name in codegen.get_nn_C_functions():
+        setattr(NestedTensor,
```

Comment:
There is no "NestedTensor.conv2d" in pytorch etc.

I don't think any of these are methods outside of maybe relu.